src/buttons/src/Button.tsx

```diff
@@ -110,10 +126,11 @@ class Button extends PureComponent<any & React.ComponentProps<typeof Box>> {
     const iconSize = theme.getIconSizeForButton(height)
 
     const pr =
-      paddingRight !== undefined ? paddingRight : Math.round(height / 2) // eslint-disable-line no-negated-condition
-    const pl = paddingLeft !== undefined ? paddingLeft : Math.round(height / 2) // eslint-disable-line no-negated-condition
+      typeof paddingRight === 'number' ? paddingRight : Math.round(height / 2)
```

Comment:
Shouldn't this also accept `string`?

Reply:
No, because of this line:
https://github.com/segmentio/evergreen/blob/db78ab2f5739aaa96e2303bbb0a47d3f64425144/src/buttons/src/Button.tsx#L139